config.rb

```diff
@@ -28,12 +28,11 @@
 data.events.each do |year, events|
   events.each do |slug, event|
     proxy "events/#{year}/#{slug}.html", "event.html", locals: { event: event, year: year, slug: slug }, ignore: true
-    if event.current
-      proxy "index.html", "current_event.html", locals: { event: event, year: year, slug: slug }, ignore: true
-    end
   end
 end
 
+page 'index.html', layout: false
```

Comment:
The landing page doesn't have any of normal layout (like the navbar)

Reply:
see how awkward this whole CMS thing is?